core/storage/user/gae_models.py

```diff
@@ -59,6 +63,14 @@ class UserSettingsModel(base_models.BaseModel):
     username = ndb.StringProperty(indexed=True)
     # Normalized username to use for duplicate-username queries. May be None.
     normalized_username = ndb.StringProperty(indexed=True)
+    # Display name for users.
```

Comment:
Perhaps move all profile-related fields to their own section after the "main user" fields, rather than putting them all over the place.